api/management-model/build.gradle.kts

```diff
@@ -54,12 +57,26 @@ openApiGenerate {
   serverVariables = mapOf("basePath" to "api/v1")
 }
 
-listOf("sourcesJar", "compileJava").forEach { task ->
-  tasks.named(task) { dependsOn("openApiGenerate") }
```

Comment:
why move this piece of code?

Reply:
I believe only the new `testImplementation` depedencies and `test { java { srcDir("src/test/java") } }` are needed. All other build script changes seem to be superfluous - please remove them to keep the scripts lean.

Reply:
Agree, this change's unrelated to the scope of the PR - please revert.